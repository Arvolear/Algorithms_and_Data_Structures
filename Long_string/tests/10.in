295734671825111617403191431948390063423727250
3437489525036531631296643018536979849