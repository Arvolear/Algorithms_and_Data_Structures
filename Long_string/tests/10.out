295734675262601142439723063245033081960707099
295734668387622092366659800651747044886747401
1016584836588937486772026948769371989041505275099342006442610886299213981722185250
86032166